Cargo.toml

```
@@ -0,0 +1,23 @@
+[package]
+name = "locked-object-store"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+thiserror = "1.0.40"
+sqlx = {version = "0.6.3", default-features = false }
+async-trait = "0.1.68"
+serde = "1.0.163"
+serde_json = "1.0.96"
```

```suggestion
thiserror = "1"
sqlx = {version = "^0.6.3", default-features = false }
async-trait = "0.1.68"
serde = "1"
serde_json = "1"
```
Since this will be a dependency, it's good to adopt as loose of versions as possible :crossed_fingers:

---

done